tests/framework/icd/physical_device.h

```diff
@@ -57,6 +58,7 @@ struct PhysicalDevice {
     // The purpose of this list is so that vkGetDeviceProcAddr returns 'a real function pointer' in tests
     // without actually implementing any of the logic inside of it.
     BUILDER_VECTOR(PhysicalDevice, VulkanFunction, known_device_functions, device_function)
+    uint32_t pci_bus{};
```

Comment:
Should use the BUILDER_VALUE macro so it can have a function automatically added. Though, since there's a constructor for it, its not a required change.

Reply:
I'll make the change anyway.